tests/test_models.py

```diff
@@ -62,8 +62,10 @@ def test_getter_setter(self):
         self.assertEqual(setting_obj.value, 'string value')
 
     def test_get_with_valid_name(self):
-        setting_value = Setting.get('TEST_SETTING_STRING')
-        self.assertEqual(setting_value, '')
+        setting_string_value = Setting.get('TEST_SETTING_STRING')
+        self.assertEqual(setting_string_value, '')
+        setting_json_value = Setting.get('TEST_SETTING_JSON')
+        self.assertEqual(setting_json_value, dict())
 
```

Comment:
Why you added this here? 
I think there should be another separated test method.